front/lib/error.ts

```diff
@@ -0,0 +1,20 @@
+export type ErrorType =
+  | "invalid_request_error"
+  | "internal_server_error"
+  | "bad_request"
```

Comment:
Errors to be refined

Reply:
What do you mean by "refined" ? the list is incomplete now?